manifests/2.4.0/opensearch-2.4.0.yml

```diff
@@ -112,8 +112,8 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
-  - name: dashboards-reports
-    repository: https://github.com/opensearch-project/dashboards-reports.git
+  - name: opensearch-reports
+    repository: https://github.com/opensearch-project/reporting.git
```

Comment:
Hi,
There is a repo split on the 2.5.0 version.
2.4.0 is untouched.
The tag exists for the dashboards-reports.
This should not be changed here.
Thanks.